src/backend/cdb/cdbllize.c

```diff
@@ -1100,6 +1100,7 @@ cdbllize_build_slice_table(PlannerInfo *root, Plan *top_plan,
 	ListCell   *lc;
 	int			sliceIndex;
 	bool		all_root_slices;
+	int			writer_slice_cnt = 0;
```

Comment:
the declaration of a variable can be moved closer to its use since we do not need compatibility with old compilers. Also, this style is already used in this function.